### ISSUE TITLE
Cannot use workDir with spaces in path

### ISSUE THREAD

**Issue by onite** [srs/gradle-node-plugin#17]
If using a workDir with spaces in the path, such as ```C:\Issue Test\work```, when trying to use an NpmTask to install, we get the error
```'C:\Issue' is not recognized as an internal or external command, operable program or batch file.```

Config and task looks as follows:
```
node {
    distBaseUrl = 'http://nodejs.org/dist'
    download = true
    workDir = file("C:/Issue Test/work")
}
task npmInstallTask(type: NpmTask) {
    args = ['install'] 
}
```

**Comment by frett27** [comment]
I Faced the same issue, 
thanks for providing this successfull plugin any way, it simplify much of our build process !!

**Comment by srs** [comment]
This is now fixed in 0.10-SNAPSHOT.

**Comment by frett27** [comment]
thumbs up, it help a lot
